### DISCUSSION TITLE
How can I enable YTUHD Tweak?

### DISCUSSION THREAD

**Discussion by Ionutovic** [YTLitePlus/YTLitePlus#183] [Q&A]
Im not able to play video at a resolution bigger than 1080p, I'm on the iPhone 14pro max.

**Comment by Balackburn** [comment]
> Im not able to play video at a resolution bigger than 1080p, I'm on the iPhone 14pro max.

Enable vp9

**Reply by Ionutovic** [reply]
> > Im not able to play video at a resolution bigger than 1080p, I'm on the iPhone 14pro max.
> 
> Enable vp9

Audio/video works with interruptions when I do that. :/

**Reply by Balackburn** [reply]
Do you have a good internet connection? Maybe the bitrate is too high for you.

**Reply by Ionutovic** [reply]
> Do you have a good internet connection? Maybe the bitrate is too high for you.

The internet was not a problem since I have gigabit fiber at home and a 5g data conexion, anyways the problem seem to be gone now for some reason. 

More details about the problem.
When I say the internet was. It causing the problem is because the video didn't stop to buffer or anything it was more like when you try to run a 4K video on a s#itty pc.

**Reply by Macleykun** [reply]
The reason for this is that you don't have the vp9 entitlements. AFAIK there's no way to get those unless you're using trollstore.
So you're effecitvly trying to render 1440p/4k withoud the efficient the VP9 gives which results in stutters if you multitask.
Source: https://github.com/PoomSmart/YTUHD?tab=readme-ov-file#sideloading
